src/wrap-methods.js

```diff
@@ -62,7 +62,7 @@ module.exports = function (integration) {
           if (mapper[method]) {
             return mapper[method].call(this, ev, this.settings)
           } else {
-            return ev.obj
+            return ev
```

Comment:
lgtm.

1. Do you know why `ev.obj` was previously returned?

2. Can you confirm this code is not currently being used? (Want to be sure not introducing regression)

Reply:
1. I think maybe the idea was to return the raw event and process them all events the same, but can't think of a good reason to not use the facade
2. No cloud mode integrations that implement a batch() method so this code path should not be traversed. However, blackbox-mock (test integration) does rely on the existing functionality, so when we bump this version in the integrations repo, we need to update the tests

Reply:
Great, thanks! 🚀